src/app/pages/send/send-crypto-asset-form/components/form-footer.tsx

```diff
@@ -28,7 +31,20 @@ export function FormFooter(props: { balance: Money }) {
       <Box mt="loose" px="loose">
         <PreviewButton />
         <SpaceBetween>
-          <Caption>Balance</Caption>
+          <Flex alignItems="center">
+            <Caption mr="tight">Available balance</Caption>
+            <Tooltip placement="top" label={balanceTooltipLabel}>
+              <Stack>
```

Comment:
Is the `Stack` needed here?

Reply:
yeah, without it tooltip is not showing, it's used so in many places, e.g. in `fee-estimate-select.layout.tsx`